docs/source/using_the_ve/virtual_ecosystem_in_use.md

```diff
@@ -129,12 +92,48 @@ The Virtual Ecosystem writes out a number of data files:
 These files are written to the standard NetCDF data file format.
 
 ```{code-cell} ipython3
+---
+editable: true
+slideshow:
+  slide_type: ''
+tags: []
+---
+# Dependencies for the data and graphing
+import pathlib
+
+import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
+import numpy as np
+import xarray
+```
+
+```{code-cell} ipython3
+---
+editable: true
+slideshow:
+  slide_type: ''
+tags: []
+---
 # Load the generated data files
 initial_state = xarray.load_dataset("/tmp/ve_example/out/initial_state.nc")
 continuous_data = xarray.load_dataset("/tmp/ve_example/out/all_continuous_data.nc")
 final_state = xarray.load_dataset("/tmp/ve_example/out/final_state.nc")
 ```
 
+```{code-cell} ipython3
+---
+editable: true
+slideshow:
+  slide_type: ''
+tags: []
+---
+# Print the name of each variable in the final state
+for key in list(final_state.keys()):
+    print(key)
+```
+
++++ {"editable": true, "slideshow": {"slide_type": ""}, "tags": []}
+
 ### Initial state and input data
 
 The `initial_state.nc` file contains all of the data required to run the model. For some
```

Comment:
In the plot below, we reshape the vector output of elevation to be a matrix of dimension 9 x 9. Can we avoid hard-coding the row and column numbers from 9 to something from the netCDF output? I imagine someone to change the grid numbers eventually so this will help to automate the plotting a bit more.

Reply:
It's actually going to need to be a new function to reverse the internal representation of cells (which is a 1D array) back onto the grid. So that's going to be a general helper function (`convert_cell_data_to_grid` or similar).

Reply:
Awesome, something like `convert_cell_data_to_grid` would be a great addition indeed. Is the grid configuration (e.g., number of rows and columns) stored anywhere in the output files?